### PR TITLE
DOC added example to reorder levels

### PR DESCRIPTION
**Pull request by ghost**
Added an example for reorder levels.

### CODE REVIEW

File: pandas/core/frame.py [hunk 1]
```diff
@@ -6837,6 +6837,31 @@ def reorder_levels(self, order: Sequence[Axis], axis: Axis = 0) -> DataFrame:
...
+        67 EEE C   Arijit
+        95 IT  A+  Tushar
+        89 ECE A     Amit
+        
+        >>> df.reorder_levels([0,2,1])
+                     name
+        70 B  CSE   Rahul
+        67 C  EEE  Arijit
+        95 A+ IT   Tushar
+        89 A  ECE    Amit
```
Comment (attack68) [inline pandas/core/frame.py:6863]: The best examples are those which are as simple and minimal as possible indicating solely the effect of the method.

```suggestion
        >>> df = pd.DataFrame({"name": ["Rahul", "Amit"]}, 
        ...         index=[[1, 2], ["A", "B"], ["Red", "Blue"]]) 
        >>> df
                      name
        1 A Red      Rahul
        2 B Blue      Amit
       
        >>> df.reorder_levels([0,2,1])
                      name
        1 Red A      Rahul
        2 Blue B      Amit
```

File: pandas/core/frame.py [hunk 2]
```diff
@@ -6837,6 +6837,27 @@ def reorder_levels(self, order: Sequence[Axis], axis: Axis = 0) -> DataFrame:
         Returns
         -------
         DataFrame
+        
+        Examples
+        --------
+        >>> df = pd.DataFrame(
+        ...  {"name" : ['Rahul','Arijit']}, 
+        ...    index = [
```
Comment (jreback) [inline pandas/core/frame.py:6845]: can you name the levels? will be much more obvious what is happening.

File: pandas/core/frame.py [hunk 3]
```diff
@@ -6837,6 +6837,27 @@ def reorder_levels(self, order: Sequence[Axis], axis: Axis = 0) -> DataFrame:
...
+        ...        ['CSE', 'EEE'],
+        ...        ['A','B']
+        ...    ]
+        ... )
+        >>> df
+                     name
+        90 CSE A    Rahul
+        77 EEE B   Arijit
+    
+        >>> df.reorder_levels([0,2,1])
```
Comment (jreback) [inline pandas/core/frame.py:6856]: use spaces between then numbers (pep)

you can see if you can enable the doc-string test for this

**Review by jreback — CHANGES_REQUESTED** [review]

File: pandas/core/frame.py (hunk 2 above)
Comment (Nikhil636) [reply pandas/core/frame.py:6845]: Do I have to give the index names in the example?

File: pandas/core/frame.py (hunk 3 above)
Comment (Nikhil636) [reply pandas/core/frame.py:6856]: ("Space between the numbers") Can you please explain this in more detail? as I am a beginner to the open-source contribution.

Comment (Nikhil636) [reply pandas/core/frame.py]: I have updated the example. Please review it.

Comment (MarcoGorelli) [reply pandas/core/frame.py]: you haven't put spaces between the numbers, please check https://www.python.org/dev/peps/pep-0008/

File: pandas/core/frame.py [hunk 4]
```diff
@@ -6837,6 +6837,32 @@ def reorder_levels(self, order: Sequence[Axis], axis: Axis = 0) -> DataFrame:
...
+        
+        Examples
+        --------
+        >>> data={'Name':['Rahul', 'Arijit', 'Tushar'],
+        ...      'Mark':[70,95,89],
+        ...      'Branch':['CSE', 'EEE', 'IT'],
+        ...      'Grade':['C', 'B', 'A'], 
+        ... }
+        >>> df=pd.DataFrame(data, columns = ['Mark', 'Branch', 'Grade', 'Name'])
+        >>> df.set_index(['Branch', 'Mark', 'Grade'], inplace = True)
```
Comment (MarcoGorelli) [inline pandas/core/frame.py:6849]: also, can you please write this as `df = df.set_index(['Branch', 'Mark', 'Grade'])`?

Comment (Nikhil636) [reply pandas/core/frame.py:6849]: Ok I will do the changes.

**Comment by MarcoGorelli** [comment]
And please fixup style errors, see https://pandas.pydata.org/pandas-docs/stable/development/contributing_codebase.html?highlight=pre%20commit#pre-commit

this is duplicative of #42935, which

**Pull request closed without merging**